pages/account/manage/event/[[...data]].js

```diff
@@ -211,9 +234,10 @@ export default function ManageEvent({ BASE_URL, event }) {
                     <Input
                       type="datetime-local"
                       name="start-date"
-                      label="Start Date"
+                      label={`Start Date${getTz(startDate)}`}
```

Comment:
Does this need a space?

```suggestion
                      label={`Start Date ${getTz(startDate)}`}
```

Reply:
`getTz` adds a space before the time zone.
I put it there so there wouldn't be an extra space wehn the time zone can't be determined because the field is incomplete.